src/NuGet.Core/NuGet.ProjectModel/PublicAPI.Shipped.txt

```diff
@@ -65,7 +65,6 @@ NuGet.ProjectModel.DependencyGraphSpec.DependencyGraphSpec() -> void
 NuGet.ProjectModel.DependencyGraphSpec.DependencyGraphSpec(Newtonsoft.Json.Linq.JObject json) -> void
 NuGet.ProjectModel.DependencyGraphSpec.DependencyGraphSpec(bool isReadOnly) -> void
 NuGet.ProjectModel.DependencyGraphSpec.GetClosure(string rootUniqueName) -> System.Collections.Generic.IReadOnlyList<NuGet.ProjectModel.PackageSpec>
-NuGet.ProjectModel.DependencyGraphSpec.GetHash() -> string
```

Comment:
As you can see, changing a method that does not take parameters to one that has an optional parameter, is a binary breaking change (it's just not breaking at compile time).

Reply:
Re-add the api to the public surface.